pkgs/servers/tt-rss/plugin-readability/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchgit }:
+
+stdenv.mkDerivation rec {
+  pname = "tt-rss-plugin-readability";
+  version = "unstable-2023-04-02";
+
+  src = fetchgit {
+    url    = "https://git.tt-rss.org/fox/ttrss-af-readability.git";
+    sha256 = "sha256-Pbwp+s4G+mOwjseiejb0gbHpInc2lvR+sv85sRP/DVg=";
+    rev = "cdc97d886cb7085f9c44a1796ee4bbbf57534d06";
+  };
+
+  installPhase = ''
+    mkdir $out
+    cp -ra * $out/
```

Comment:
```suggestion
    cp -a * $out/
```

-a implies -r

Reply:
indeed. I cargo-culted it from some other places in nixpkgs 😬